package.json

```diff
@@ -97,7 +97,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
-    "@folio/stripes-acq-components": "^2.1.0",
+    "@folio/stripes-acq-components": "^2.3.3000482",
```

Comment:
? `^` uses lates minor and patch

Reply:
Thanks!
Reverted the irrelevant version bumps - my experiments to make linting work